releaseUtils/build.js

```diff
@@ -1,9 +1,13 @@
 /* eslint-disable import/no-commonjs, global-require */
 const execa = require('execa');
 
-const build = buildDir =>
+const build = (buildDir, tsTemp) =>
     execa.shell(
-        `npx babel --ignore __benchmarks__,__tests__ ./src -d ${buildDir}`
-    );
+        `tsc -p tsconfig.json`
```

Comment:
We should add a eslint config here, there is no need for backticks

Reply:
I think it's better to do this in a separate task